app/src/main/java/app/revanced/integrations/boostforreddit/FixSLinksPatch.java

```diff
@@ -0,0 +1,93 @@
+package app.revanced.integrations.boostforreddit;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.os.StrictMode;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import app.revanced.integrations.shared.Logger;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.SocketTimeoutException;
+import java.net.URL;
+
+public final class FixSLinksPatch {
+    public static String getUserAccessToken(Context context) {
```

Comment:
Should be private

Reply:
Should it still be private? Interfaces don't allow private+default